docs/Ecosystem.md

```diff
@@ -42,9 +42,9 @@ See the [Data Provider](./DataProviders.md#available-providers) page.
 
 ## Miscellaneous
 
-- [marmelab/ra-realtime](https://github.com/marmelab/react-admin/tree/master/packages/ra-realtime): enable realtime updates
-- [marmelab/ra-tree-ui-materialui](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-ui-materialui/): Components to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
-- [marmelab/ra-tree-core](https://github.com/marmelab/react-admin/blob/master/packages/ra-tree-core/): Components providing the logic but no UI to show data represented as a tree. This package is part of our [Labs](/Labs.md) experimentations. This means it misses some features and might not handle all corner cases. Use it at your own risks. Besides, we would really appreciate some feedback!
+- [marmelab/ra-realtime](https://github.com/marmelab/react-admin/tree/2.x/packages/ra-realtime): enable realtime updates
```

Comment:
Shouldn't we add a note indicating its for v2 only ?

Reply:
sorry, this comment should have come first

Reply:
it's the v2 documentation. If we start speaking about v3 in the v2 docs, we're in for a long and hard work!

Reply:
Ah sorry I missed the target branch :ok_hand: